src/Resource.hs

```diff
@@ -42,40 +50,63 @@ data Monitor = Monitor { monitorName :: T.Text
 instance Show Monitor where
   show m = "\"" ++ T.unpack (monitorName m) ++ "\""
 
+queryHasThreshold :: T.Text -> Bool
+queryHasThreshold t
+  | T.isSuffixOf "count_by_status()" t = False
+  | T.isSuffixOf "pct_by_status()" t = False
+  | T.isPrefixOf "events(" t = True
```

Comment:
technically, having this branch here doesn't do anything (since otherwise = True), but I guess it's good documentation of the intention?